docs/xcm/building-with-xcm/xc-reserve-transfer.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 7
+---
+
+# Reserve Transfer Assets
+
+XCM allows us to transfer assets from one chain to another in several ways. In this chapter we'll see how to use reserve transfers.
+
+Suppose we want Shibuya `SBY` tokens to be present on Shiden as a wrapped asset `wSBY`.
+
+- Shibuya chain will need to have Shiden's sovereign account. This account is controlled by Shiden and would represent funds that were sent to the remote chain (Shiden from Shibuya's perspective)
+- Shiden should create `wSBY` asset and configure it to act as a cross-chain asset
+- HRMP channels should also be configured for chains to communicate and exchange XCM messages
+- To pay for execution time `wSBY` should be configured as a payment asset on Shiden
+
+During the actual transfer the following happens:
+1. Some `SBY`s are moved from source account to the sovereign account of Shiden on Shibuya
+2. An XCM containing `ReserveTransferAssets` instruction is sent to Shiden
+3. That message is processed by the `assets` pallet on Shiden, the corresponding amount of `wSBY`s is minted on Shiden
+4. Minted `wSBY` tokens are deposited to the destination account
+5. Some amount is deducted as a payment for execution time
+
+**Note:** Please keep in mind that everything above is just an example specific to the implementation of two particular parachains. XCM does not dictate or impose any restrictions on how to interpret incoming messages, or how to manage derivative assets. Other parachains may or may not use `assets` pallet and technically the only thing we can say for sure is that `assets_reserve_transfer` will form an XCM that would be sent to a remote chain specified by its `parachain_id`. Everything else is dependent on the remote chain and its logic.
```

Comment:
How about having the same _story_ for assets withdraw and reserve transfer?

They are connected, basically one is the inverse operation of the other.